src/FINDOLOGIC/Export/CSV/CSVExporter.php

```diff
@@ -56,7 +56,8 @@ public function serializeItemsToFile(
         $csvString = $this->serializeItems($items, $start, $count, $total);
         $targetPath = sprintf('%s/findologic.csv', $targetDirectory);
 
-        file_put_contents($targetPath, $csvString);
+        // Clear CSV contents if a new export starts with start 0. Otherwise append the contents.
```

Comment:
Comment describes what the line does (which you can tell from the line itself). Better use it to explain why the line does what it does.

Reply:
:+1: